src/v3/pages/reports/meeting_attendance/index.styles.tsx

```diff
@@ -0,0 +1,55 @@
+import { Box, Divider, Typography } from '@mui/material';
+import TextField from '@components/textfield';
+import { styled } from '@mui/system';
+
+export const StyledContentBox = styled(Box)({
+  display: 'flex',
+  gap: '16px',
+  flexDirection: 'column',
+});
+
+export const StyledBox = styled(Box)(({ row }: { row: boolean }) => ({
```

Comment:
@Logan08691: please fix this code to prevent the Invalid prop to a DOM error. Thanks.